### PR TITLE
 fix russian translation

### PR DESCRIPTION
**Pull request by Anime-pdf**
## Checklist

- [ ] Tested the change ingame
- [ ] Provided screenshots if it is a visual change
- [ ] Tested in combination with possibly related configuration options
- [ ] Written a unit test if it works standalone, system.c especially
- [ ] Considered possible null pointers and out of bounds array indexing
- [ ] Changed no physics that affect existing maps
- [ ] Tested the change with [ASan+UBSan or valgrind's memcheck](https://github.com/ddnet/ddnet/#using-addresssanitizer--undefinedbehavioursanitizer-or-valgrinds-memcheck) (optional)

### CODE REVIEW

**Comment by def-** [comment]
@gerdoe-jr can you review this please?

File: data/languages/russian.txt [hunk 1]
```diff
@@ -1297,31 +1298,31 @@ CHN
 == CHN
 
 Getting server list from master server
-== Получение списка серверов с главного сервера
+== Получение списка серверов от мастер-сервера
 
 Leak IP
 == Передать IP
```
Comment (gerdoe-jr) [inline data/languages/russian.txt:1304]: `Передавать IP`

File: data/languages/russian.txt [hunk 2]
```diff
@@ -1297,31 +1298,31 @@ CHN
...
-== Используйте новый OpenGL
+== Использовать новый OpenGL
 
 Hookline
-== Линия крюка
+== 
 
 No hit
-== Нет удара
+==
```
Comment (gerdoe-jr) [inline data/languages/russian.txt:1322]: Предлагаю `Нет сцепления`, но звучит глупо.

File: data/languages/russian.txt [hunk 3]
```diff
@@ -1297,31 +1298,31 @@ CHN
...
-== Линия крюка
+== 
 
 No hit
-== Нет удара
+== 
 
 Hookable
-== Можно крючком
+==
```
Comment (gerdoe-jr) [inline data/languages/russian.txt:1325]: Может, `Есть сцепление`, но звучит также глупо.

**Review by gerdoe-jr — CHANGES_REQUESTED** [review]

File: data/languages/russian.txt (hunk 3 above)
Comment (Anime-pdf) [reply data/languages/russian.txt:1325]: да, лучше подождать свежей мысли

File: data/languages/russian.txt (hunk 2 above)
Comment (Anime-pdf) [reply data/languages/russian.txt:1322]: Да, глуповато 😄

File: data/languages/russian.txt (hunk 1 above)
Comment (Anime-pdf) [reply data/languages/russian.txt]: Исправил

**Pull request merged**